README.md

```diff
@@ -4,6 +4,12 @@ This package provides Julia bindings to the [FFTW](http://www.fftw.org/) library
 fast Fourier transforms, as well as functionality useful for signal processing.
 These functions were formerly a part of Base Julia.
 
+The FFTW library will be downloaded on versions of Julia where it is no longer distributed
+as part of Julia.
+Note that FFTW is licensed under GPLv2 or higher (see
+[its license file](http://www.fftw.org/doc/License-and-Copyright.html)), but the bindings
+here are licensed under MIT.
```

Comment:
This is a bit unclear.   Best to spell out the consequences:
```
This means that code using the FFTW library via the FFTW.jl bindings is subject to FFTW's licensing terms.  Code using alternative implementations of the FFTW API, such as [MKL's FFTW3 interface](https://software.intel.com/en-us/mkl-developer-reference-c-fftw3-interface-to-intel-math-kernel-library) are instead subject to the alternative's license.
```

Reply:
Ah yes, MKL deserves a mention here, but I'm unclear on what "code using the FFTW library via the FFTW.jl bindings is subject to FFTW's licensing terms" means.

Reply:
If you distribute a derived/combined work, i.e. a program that links to FFTW, then that distribution falls under the terms of the GPL (e.g. must include complete source code etc.)

If you just distribute source code that links to FFTW.jl, and users have to download FFTW or MKL to provide the backend, then the GPL probably doesn't have much effect on you.